OpenOversight/app/main/views.py

```diff
@@ -251,6 +257,29 @@ def add_department():
         return render_template('add_department.html', form=form)
 
 
+@main.route('/department/<int:department_id>')
+def list_officer(department_id, page=1, from_search=False):
+    if request.args.get('page'):
+        page = int(request.args.get('page'))
+
+    if request.args.get('from_search'):
+        from_search = bool(request.args.get('from_search'))
```

Comment:
Hmm, I'm definitely missing something - I went to merge this tonight - I saw the behavior of using `from_search` with pagination for `from_search=True` totally works as expected. However, when I use the "Browse" flow:

<img width="1004" alt="screen shot 2018-05-02 at 10 13 19 pm" src="https://user-images.githubusercontent.com/7832803/39560822-276fcae2-4e56-11e8-9fc3-6aff19968683.png">

the first page is displayed as expected (without "No Results Found"), but for pages 2 and beyond the links have `from_search=True` so they display "No Results Found"